solr/solr-ref-guide/modules/deployment-guide/pages/solrj.adoc

```diff
@@ -136,7 +136,7 @@ If not an `IllegalArgumentException` will be thrown.
 
 ==== Base URLs of CloudSolrClient
 
-It is also possible to specify base URLs for `CloudSolrClient`, but URLs are expected to point to the root Solr path (e.g., `\http://hostname:8983/solr`).
+It is also possible to specify base URLs for `CloudSolrClient`, but URLs must point to the root Solr path (e.g., `\http://hostname:8983/solr`).
```

Comment:
Need the leading slash?

Reply:
Yes we do, the leading slash ensures that asciidoc renders the URL as unclick-able.  (i.e. we want the URL to be formatted as code, not an actual link that a user would get value in clicking).  See some of the other URLs in this page as examples...